whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/IrtSpec.groovy

```diff
@@ -459,9 +459,7 @@ class IrtSpec extends BaseQueryUpdateSpec {
         ack.countErrorWarnInfo(1, 0, 0)
         ack.errors.any { it.operation == "Create" && it.key == "[irt] irt-test" }
         ack.errorMessagesFor("Create", "[irt] irt-test") ==
-                ["\"abuse-mailbox:\" can only be added to ROLE objects intended to be " +
-                 "referenced through the \"abuse-c:\" attribute in ORGANISATION, " +
-                 "INET(6)NUM and AUT-NUM objects."]
+                ["\"abuse-mailbox\" is not valid for this object type"]
```

Comment:
More specific error message removed?

Reply:
The abuse-mailbox attribute is now no longer allowed on IRT objects, so the error message is correct in my opinion..

Reply:
Ok! It's the generic error message for any invalid attribute.

Reply:
Yes, we could say that maybe the test is no longer necessary as there's no point in testing all the attributes that are not allowed on the object, but seen as abuse-mailbox was allowed before but will no longer be allowed I think it's good to have captured this change in behaviour for regression test purposes..